packages/compass-e2e-tests/tests/collection-aggregations-tab.test.ts

```diff
@@ -501,116 +536,25 @@ describe('Collection aggregations tab', function () {
-    });
-
-    after(async function () {
-      process.env.COMPASS_SHOW_NEW_AGGREGATION_TOOLBAR =
-        initialAggregationToolbarValue;
-      process.env.COMPASS_ENABLE_AGGREGATION_EXPLAIN =
-        initialAggregationExplainValue;
-      await afterTests(compass, this.currentTest);
-    });
-
-    afterEach(async function () {
-      await afterTest(compass, this.currentTest);
-    });
-
-    it('shows the explain for a pipeline', async function () {
-      await browser.clickVisible(Selectors.AggregationExplainButton);
-      await browser.waitForAnimations(Selectors.AggregationExplainModal);
-
-      const modal = await browser.$(Selectors.AggregationExplainModal);
-      await modal.waitForDisplayed();
-      await browser.waitForAnimations(Selectors.AggregationExplainModal);
-      expect(await modal.getText()).to.contain('Query Performance Summary');
-    });
-  });
-
-  async function goToRunAggregation() {
-    if (await browser.$(Selectors.AggregationBuilderWorkspace).isDisplayed()) {
-      await browser.clickVisible(Selectors.RunPipelineButton);
-    }
-    const resultsWorkspace = await browser.$(
-      Selectors.AggregationResultsWorkspace
-    );
-    await resultsWorkspace.waitForDisplayed();
-  }
-
-  async function goToEditPipeline() {
-    if (await browser.$(Selectors.AggregationResultsWorkspace).isDisplayed()) {
-      await browser.clickVisible(Selectors.EditPipelineButton);
-    }
-    const builderWorkspace = await browser.$(
-      Selectors.AggregationBuilderWorkspace
-    );
-    await builderWorkspace.waitForDisplayed();
-  }
-
-  async function getDocuments() {
-    // Switch to JSON view so it's easier to get document value
-    await browser.clickVisible(
-      Selectors.AggregationResultsJSONListSwitchButton
-    );
-    // Get all visible documents
-    const documents = await browser.$$(Selectors.DocumentJSONEntry);
-    // Get ace editor content and parse it
-    const parsed = await Promise.all(
-      documents.map(async (doc) => {
-        const aceEditor = await doc.$('.ace_content');
-        return JSON.parse(await aceEditor.getText());
-      })
-    );
-    return parsed;
-  }
-
   it('supports running and editing aggregation', async function () {
     // Set first stage to match
     await browser.focusStageOperator(0);
     await browser.selectStageOperator(0, '$match');
     await browser.setAceValue(Selectors.stageEditor(0), '{ i: 5 }');
 
     // Run and wait for results
-    await goToRunAggregation();
+    await goToRunAggregation(browser);
 
     // Get all documents from the current results page
-    const docs = await getDocuments();
+    const docs = await getDocuments(browser);
 
     expect(docs).to.have.lengthOf(1);
     expect(docs[0]).to.have.property('_id');
     expect(docs[0]).to.have.property('i', 5);
     expect(docs[0]).to.have.property('j', 0);
 
     // Go back to the pipeline builder
-    await goToEditPipeline();
+    await goToEditPipeline(browser);
```

Comment:
question: what does this change do? is it related to the change of port?

Reply:
I moved `goToEditPipeline` out of the scope where `browser` was defined. I also moved one test suite out, because they used the same variables' names, and was hard to track if the value wasn't overwritten at some point.